code-reviews/example1/styles.css

```diff
@@ -0,0 +1,15 @@
+.main {
+    background-color: #add8e6;
+    color: black;
+    padding: 12px;
+}
+
+.list {
+    font-weight: bold;
+    margin-bottom: 4;
+}
+
+recipes {
```

Comment:
The class recipe has not been referenced well:I think it should be .recipes ,The dot was not included